zipkin/src/main/java/com/linecorp/armeria/internal/tracing/LogRequestContextWarningOnce.java

```diff
@@ -0,0 +1,36 @@
+package com.linecorp.armeria.internal.tracing;
+
+import java.util.function.Supplier;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.common.tracing.RequestContextCurrentTraceContext;
+
+public enum LogRequestContextWarningOnce implements Supplier<RequestContext> {
```

Comment:
- Could be package-local
- Given this is only used by `RequestContextCurrentTraceContext`, how about moving this into it?